scripts.js

```diff
@@ -0,0 +1,120 @@
+var newGameBtn = document.getElementById('js-newGameButton');
```

Comment:
clean your code

at first declare variables, next declare functions, next add event listeners and next call your methods ;)